relay/config.go

```diff
@@ -18,6 +18,7 @@ import (
 const (
 	DefaultDialTimeout                 = 20 // seconds
 	DefaultMessageAggregationBatchSize = 8
+	DefaultChunkSize                   = 4*1024*1024 - 1024
```

Comment:
@yoshidan  If there is a reason for this default size, please let me know. I am curious why it is not 4MB.

Reply:
An error occurred when the actual size was exactly 4 MB. If the max header size is set to 4 MB, the actual message size exceeds 4 MB.

```go
	header := [4 * 1024 * 1024]byte{}
	chunk := elc.MsgUpdateClientStreamChunk{
		Chunk: &elc.MsgUpdateClientStreamChunk_HeaderChunk{
			HeaderChunk: &elc.UpdateClientStreamHeaderChunk{
				Data: header[:],
			},
		},
	}
	message, err := chunk.Marshal()
	if err != nil {
		t.Errorf("Failed to marshal chunk: %v", err)
		return
	}
	println(len(message), len(header))
```
output is 4194314 4194304


There is no particular reason for it to be 1024; it can be a little smaller.

Reply:
Indeed, it is necessary to consider the size of protobuf metadata.

Reply:
Could you provide that context as a code comment shortly?
In addition, could you change the name `DefaultChunkSize` to `DefaultMaxChunkSize`?

Reply:
Fixed by https://github.com/datachainlab/lcp-go/pull/49/commits/1896b1dd39ce471f65ac883b1a75ee23cdb04897